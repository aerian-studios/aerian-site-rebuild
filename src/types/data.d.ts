import * as React from "react";

export const isImageSharp = (
    image: ImageSharp | string
): image is ImageSharp => {
    return typeof image !== "string";
};

export interface Image {
    alt?: string;
    image: string | ImageSharp;
}

export interface ImageSharp {
    childImageSharp: {
        sizes: ImageSharpSizes;
        src: string;
    };
}

export interface ImageSharpSizes {
    aspectRatio?: number;
    sizes?: string;
    src: string;
    srcSet?: string;
    tracedSVG?: string;
}
