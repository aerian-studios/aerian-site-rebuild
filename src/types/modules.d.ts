declare module "*.svg";
declare module "*.png";
declare module "*.json";

// For storyshots
declare module "babel-plugin-require-context-hook/register";

// for maps
declare module "pigeon-overlay";
declare module "pigeon-maps";
