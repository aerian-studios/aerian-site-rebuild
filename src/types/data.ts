import { Stats } from "fs";
import { ParsedPath } from "path";

export type ImageField = string | ImageSharp;
export type FileField = string | Partial<FileNode>;

// Images

export interface GalleryImage {
    alt?: string;
    image: ImageField;
}

export interface FileNode extends ParsedPath, Stats {
    publicURL: string;
    id: string;
    children: any[];
    sourceInstanceName: string;
    absolutePath: string;
    relativePath: string;
    relativeDirectory: string;
    extension: string;
    size: number;
    prettySize: string;

    modifiedTime: string;
    accessTime: string;
    changeTime: string;
    birthTime: string;
    internal: {
        contentDigest: string;
        type: "Directory" | "File";
        mediaType?: string;
        description: string;
    };
}

export interface ImageSharp extends Partial<FileNode> {
    childImageSharp: {
        fluid?: ImageSharpFluid;
        fixed?: ImageSharpFixed;
    };
}
export interface ImageSharpFluid {
    aspectRatio?: number;
    sizes?: string;
    src: string;
    srcSet?: string;
    tracedSVG?: string;
    base64?: string;
}

export interface ImageSharpFixed {
    aspectRatio?: number;
    src: string;
    srcSet?: string;
    tracedSVG?: string;
    base64?: string;
    width: number;
    height: number;
}

// PAGES

export interface NodeList<T> {
    edges: Array<Edge<T>>;
}

export interface Edge<T> {
    node: T;
}
export interface PagesListData {
    data: {
        allPagesJson: PageList;
    };
}

export interface PageList {
    edges: PageListNode[];
}

export type PageListNode = Edge<PageRef>;

export interface PageRef {
    id: string;
    path: string;
    title: string;
}

export interface PageSection {
    title: string;
    image?: ImageField;
    smallImage?: ImageField;
    subtitle: string;
    blurb: string;
    activities?: Activity[];
    successStory?: SuccessStory;
    testimonial?: Testimonial;
}

export interface Page {
    title: string;
    path: string;
    heroImage?: ImageField;
    subheading: string;
    description: string;
    seoTitle: string;
    seoDescription: string;
    seoKeywords: string;
}

export interface WhatWeDo extends Page {
    sections?: PageSection[];
}

export interface MeetTheTeam extends Page {
    staff: Staff[];
}

export interface About extends Page {
    infographic?: Infographic[];
}

export type InfographicType =
    | "Single Line"
    | "Compact Single Line"
    | "Split"
    | "Bar"
    | "Vertical"
    | "Horizontal"
    | "Surround";
export interface Infographic {
    primaryText?: string;
    type: InfographicType;
    secondaryText?: string;
    image?: ImageField;
    imageCount: number;
    secondaryImage?: ImageField;
}

export interface Staff {
    name: string;
    jobTitle: string;
    live: boolean;
    imageNormal: ImageField;
    imageFunny: ImageField;
    description: string;
    fact: string;
    skills: string[];
}

export interface ReactRouterLocation {
    action: string;
    hash: string;
    key: string;
    pathname: string;
    search: string;
    state: null | string;
}

export interface Testimonial {
    quote?: string;
    person?: string;
    title?: string;
    avatar?: ImageField;
    image?: ImageField;
}

export interface Stat {
    title: string;
    text: string;
    image: ImageField;
}
export interface SuccessStory {
    title: string;
    stats?: Stat[];
}

export interface Activity {
    title: string;
    text: string;
}

export interface Hurdle {
    title: string;
    image: ImageField;
    text: string;
}

export interface ProjectStage {
    description: string;
    hurdles: Hurdle[];
}

export interface Performance {
    title: string;
    text: string;
}
export interface ProjectBox {
    name: string;
    slug: string;
    heroImage: ImageField;
    thumbnail: ImageField;
    client: Client;
    titleLineOne: string;
}
export interface Project {
    name: string;
    slug: string;
    heroImage: ImageField;
    heroVideo?: string;
    titleLineOne: string;
    titleLineTwo?: string;
    client: Client;
    goLiveDate?: string;
    caseStudyTitle: string;
    caseStudyText: string;
    caseStudyImage: ImageField;
    externalUrl: string;
    rolloverDetails: string;
    homepage: boolean;
    featured: boolean;
    gallery: GalleryImage[];
    challenge: ProjectStage;
    solution: ProjectStage;
    results: ProjectStage;
    performance?: Performance[];
    testimonial?: Testimonial;
}

export interface Client {
    name: string;
    logo: ImageField;
    promoLogo: ImageField;
    featured: boolean;
    featuredProject?: string;
    id: string;
}
