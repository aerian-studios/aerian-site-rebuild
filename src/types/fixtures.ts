import {
    About,
    Client,
    ImageField,
    ImageSharp,
    Infographic,
    MeetTheTeam,
    PageSection,
    Performance,
    Project,
    ProjectBox,
    SuccessStory,
    Testimonial,
    WhatWeDo
} from "./data";

import * as aboutData from "../pages/about.json";
import * as clientData from "../pages/clients/cnn.json";
import * as meetTheTeamData from "../pages/meet-the-team.json";
import * as projectData from "../pages/projects/bbc-iwonder-perfect-diet-for-you.json";
import * as whatWeDoData from "../pages/what-we-do.json";

import { absolutifyURLs } from "../lib/helpers";
export const client = absolutifyURLs<Client>(clientData);

export const project = { ...absolutifyURLs<Project>(projectData), client };
export const projectBox: ProjectBox = {
    name: project.name,
    slug: project.slug,
    titleLineOne: project.titleLineOne,
    thumbnail: project.heroImage,
    heroImage: project.heroImage,
    client
};
export const whatWeDo = absolutifyURLs<Required<WhatWeDo>>(whatWeDoData);
export const about = absolutifyURLs<About>(aboutData);
export const meetTheTeam = absolutifyURLs<MeetTheTeam>(meetTheTeamData);
export const projectStage = project.challenge;
export const hurdles = projectStage.hurdles;
export const hurdle = hurdles[0];
export const performances: Performance[] = project.performance as Performance[];
export const performance = performances[0];
export const testimonial: Testimonial = project.testimonial as Testimonial;
export const testimonialSection: PageSection = whatWeDo.sections[0];
export const successStorySection: PageSection = whatWeDo.sections[1];
export const successStory: SuccessStory = successStorySection.successStory as SuccessStory;
export const pageSection = {
    ...successStorySection,
    testimonial: testimonialSection.testimonial
} as Required<PageSection>;
export const activity = pageSection.activities[0];
export const stat = successStory.stats![0];
export const staff = meetTheTeam.staff;
export const person = staff[0];
export const infographics = about.infographic as Infographic[];
export const infographic = infographics[0];

export const image: string = person.imageFunny as string;
export const imageSharpFluid: ImageSharp = {
    childImageSharp: {
        fluid: {
            aspectRatio: 3 / 4,
            sizes: "(max-width: 290px) 100vw, 290px",
            base64:
                "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAYCAYAAAD6S912AAAACXBIWXMAAAsSAAALEgHS3X78AAAEe0lEQVQ4y42Te1BUVRzH70z/9V9ZToXy2GXfD3Z57C4sDyHtreWMIqmEgUVCak4qwiiRiausiIhk6JLiAzXGEE1FBVMaEBtsrJnQwsEaNTMbmxRrFJZP516EWXykZ+Y3d/fc7/mc7+9xJQJWf3+/8vT7/dy9BvdkzaDufkt6EKznxnXKS0uo+rSCP69eVfb6+vqGaf8XGCjs6enhtZdeRJIkHn9MwmGzcv589yNBhzns7e1VfnuKP2GcTY8nO51Vs96iaNp4ctImcvn3K0NZPBJQXn9fv8EHGWkcKfuIrrpqftmzlXO1VVTNnUG1SD/Q5QOBMszvHxB9297G4sxUjlV5+bqylK76Lzi2bjUNxflsWeOhr1858HCH/ju3fnP8KMU502mqWMmmRR/Sss7L/pKl1OTPY/daDzf/+ffhNRxwONDd002H2LDofVrWe2kuW8FRz8ecKPPQXuGhtVwAL10cAN5ntIYBBwW/7d9Hw7ICftjpo2tLJY1FBTQVL6FzUyXt4oKe7u6HAwMFt8SBI6XLOSAcndpcSUt5CQ3zFnCgpJDmSi83/7r2aCkHujwrGrPLs4RD65bTurGcVpHyHs9CTjTsHLr8QV/M8MG+A7z2xxWqlxbgK5zHcd9qmtcuY33eLH463aG87+29PQS7+ykF3jRws+j47Vs0VJRQNieLls1llOZm8NnC9zjbcXKYG3ke5Qgc9CHg4At5nTzZQXVRPnVF8+ms89HiK6XN52V1QR4rSsro7Dxzz3APQiX5RWAtjhxuIjnlZTInTaamYDY/1lXx84FaOr+spjAnmxFPP4fZFMHENyZTXLySxoOHuXDh4r01bGtrJzPzXWKi43A43FgskazJn0tn/Qa6m3fzff1WslJT0WgNGAwmgoPDGD0qjHC1AacjnqKiZcK1H2n79h1kZ+cSGelArzPhcsUTF5vAE089S17GJM41bKTr8C6+27uT1PGvowrXodHIoUevN2G1RmI22xUjly5dRgoOVhEaqhab1jsRgcVqEw5C8eXN5PzeDZxp3MHBmhqy3pxKiNBqwrWo1Vq0wq3ZbBPQKGEiidraXUgGgxk5jEazkoocJgFOco/h88X5/LpvG6f2bmP/1t3MnpZOmFqDTmdU3JlMVgVoMgmoJUqUKh5pVFCIIpChOhFarR6bLYqEhOdxu8eSnTaDqhVeWr86Rm5WNuEiXaPJImpsE7oY7CJkt0aDlVhXItKIJ0diMloYHaLmnSmT8M6fI+oRS0J8ihIOIYpzp7AwdwFZU6djEFr5Qrs9huiYWHE2QpTAIHrgxBHjRhopxkCl1vNCUhKN61fx9pQ0YoT1ROEw3p3MmKRxxMYmKk6S3fHY7NFKA+WQG6JSaZS0o6JcRAsj0jMjgwgarSLt1VdYPDODUDEGcoFdrgSRdgqJiWPR6S2kTUwlfcIELBGRwkmccOhQQGGh4ei0RsVhpN2JFBoSTliYFrslgmSnUznsFHPociYoMLkuKpWWKJGiS4BstmglXbmzBoOFoKBgAdUQIf7L8R+bum8HTMuX7wAAAABJRU5ErkJggg==",
            src: image,
            srcSet: image
        }
    }
};

export const imageSharpFixed: ImageSharp = {
    childImageSharp: {
        fixed: {
            width: 290,
            height: 290,
            base64:
                "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAYCAYAAAD6S912AAAACXBIWXMAAAsSAAALEgHS3X78AAAEe0lEQVQ4y42Te1BUVRzH70z/9V9ZToXy2GXfD3Z57C4sDyHtreWMIqmEgUVCak4qwiiRiausiIhk6JLiAzXGEE1FBVMaEBtsrJnQwsEaNTMbmxRrFJZP516EWXykZ+Y3d/fc7/mc7+9xJQJWf3+/8vT7/dy9BvdkzaDufkt6EKznxnXKS0uo+rSCP69eVfb6+vqGaf8XGCjs6enhtZdeRJIkHn9MwmGzcv589yNBhzns7e1VfnuKP2GcTY8nO51Vs96iaNp4ctImcvn3K0NZPBJQXn9fv8EHGWkcKfuIrrpqftmzlXO1VVTNnUG1SD/Q5QOBMszvHxB9297G4sxUjlV5+bqylK76Lzi2bjUNxflsWeOhr1858HCH/ju3fnP8KMU502mqWMmmRR/Sss7L/pKl1OTPY/daDzf/+ffhNRxwONDd002H2LDofVrWe2kuW8FRz8ecKPPQXuGhtVwAL10cAN5ntIYBBwW/7d9Hw7ICftjpo2tLJY1FBTQVL6FzUyXt4oKe7u6HAwMFt8SBI6XLOSAcndpcSUt5CQ3zFnCgpJDmSi83/7r2aCkHujwrGrPLs4RD65bTurGcVpHyHs9CTjTsHLr8QV/M8MG+A7z2xxWqlxbgK5zHcd9qmtcuY33eLH463aG87+29PQS7+ykF3jRws+j47Vs0VJRQNieLls1llOZm8NnC9zjbcXKYG3ke5Qgc9CHg4At5nTzZQXVRPnVF8+ms89HiK6XN52V1QR4rSsro7Dxzz3APQiX5RWAtjhxuIjnlZTInTaamYDY/1lXx84FaOr+spjAnmxFPP4fZFMHENyZTXLySxoOHuXDh4r01bGtrJzPzXWKi43A43FgskazJn0tn/Qa6m3fzff1WslJT0WgNGAwmgoPDGD0qjHC1AacjnqKiZcK1H2n79h1kZ+cSGelArzPhcsUTF5vAE089S17GJM41bKTr8C6+27uT1PGvowrXodHIoUevN2G1RmI22xUjly5dRgoOVhEaqhab1jsRgcVqEw5C8eXN5PzeDZxp3MHBmhqy3pxKiNBqwrWo1Vq0wq3ZbBPQKGEiidraXUgGgxk5jEazkoocJgFOco/h88X5/LpvG6f2bmP/1t3MnpZOmFqDTmdU3JlMVgVoMgmoJUqUKh5pVFCIIpChOhFarR6bLYqEhOdxu8eSnTaDqhVeWr86Rm5WNuEiXaPJImpsE7oY7CJkt0aDlVhXItKIJ0diMloYHaLmnSmT8M6fI+oRS0J8ihIOIYpzp7AwdwFZU6djEFr5Qrs9huiYWHE2QpTAIHrgxBHjRhopxkCl1vNCUhKN61fx9pQ0YoT1ROEw3p3MmKRxxMYmKk6S3fHY7NFKA+WQG6JSaZS0o6JcRAsj0jMjgwgarSLt1VdYPDODUDEGcoFdrgSRdgqJiWPR6S2kTUwlfcIELBGRwkmccOhQQGGh4ei0RsVhpN2JFBoSTliYFrslgmSnUznsFHPociYoMLkuKpWWKJGiS4BstmglXbmzBoOFoKBgAdUQIf7L8R+bum8HTMuX7wAAAABJRU5ErkJggg==",
            src: image,
            srcSet: image
        }
    }
};
export const imageSharp = imageSharpFluid;
export const imageField: ImageField = imageSharp;
