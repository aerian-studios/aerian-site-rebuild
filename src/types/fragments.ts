import { graphql } from "gatsby";

export const pageFieldsFragment = graphql`
    fragment PageFields on PagesJson {
        title
        heroImage {
            childImageSharp {
                fluid(srcSetBreakpoints: [1024, 720, 360]) {
                    ...GatsbyImageSharpFluid_withWebp
                }
            }
        }
        subheading
        description
        seoTitle
        seoDescription
        seoKeywords
    }
`;

export const clientFragment = graphql`
    fragment Client on ClientsJson {
        name
        logo {
            childImageSharp {
                fluid(maxWidth: 600, srcSetBreakpoints: [420, 210]) {
                    ...GatsbyImageSharpFluid_withWebp_tracedSVG
                }
            }
            publicURL
        }
        promoLogo {
            childImageSharp {
                fluid(maxWidth: 600, srcSetBreakpoints: [420, 210]) {
                    ...GatsbyImageSharpFluid_withWebp_tracedSVG
                }
            }
            publicURL
        }
        featured
        featuredProject
    }
`;

export const projectBoxFragment = graphql`
    fragment ProjectBox on ProjectsJson {
        name
        slug
        titleLineOne
        client {
            ...Client
        }
        heroImage {
            childImageSharp {
                fluid(
                    maxWidth: 1066
                    maxHeight: 1066
                    srcSetBreakpoints: [1066, 720, 360, 250]
                ) {
                    ...GatsbyImageSharpFluid_withWebp_noBase64
                }
            }
            publicURL
        }
    }
`;

export const projectFragment = graphql`
    fragment Project on ProjectsJson {
        name
        slug
        client {
            ...Client
        }
        titleLineOne
        titleLineTwo
        goLiveDate
        caseStudyTitle
        caseStudyText
        externalUrl
        heroImage {
            childImageSharp {
                fluid(srcSetBreakpoints: [1024, 720, 360]) {
                    ...GatsbyImageSharpFluid_withWebp
                }
            }
            publicURL
        }
        heroVideo {
            publicURL
        }
        gallery {
            image {
                childImageSharp {
                    fluid(srcSetBreakpoints: [720, 360]) {
                        ...GatsbyImageSharpFluid_withWebp
                    }
                }
                publicURL
            }
            alt
        }
        challenge {
            description
            hurdles {
                title
                image {
                    childImageSharp {
                        fixed(width: 290) {
                            ...GatsbyImageSharpFixed_withWebp_tracedSVG
                        }
                    }
                    publicURL
                }
                text
            }
        }
        solution {
            hurdles {
                title
                image {
                    childImageSharp {
                        fixed(width: 290) {
                            ...GatsbyImageSharpFixed_withWebp_tracedSVG
                        }
                    }
                    publicURL
                }
                text
            }
            description
        }
        results {
            description
            hurdles {
                title
                image {
                    childImageSharp {
                        fixed(width: 290) {
                            ...GatsbyImageSharpFixed_withWebp_tracedSVG
                        }
                    }
                    publicURL
                }
                text
            }
        }
        performance {
            title
            text
        }
        testimonial {
            quote
            person
            title
            avatar {
                childImageSharp {
                    fixed(width: 75) {
                        ...GatsbyImageSharpFixed_withWebp_tracedSVG
                    }
                }
                publicURL
            }
        }
        id
    }
`;
