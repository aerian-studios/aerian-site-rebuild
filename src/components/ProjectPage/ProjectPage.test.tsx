import * as React from "react";

import * as renderer from "react-test-renderer";

import { project, projects } from "../../types/fixtures";
import { ProjectPage } from "./index";

describe("ProjectPage", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(<ProjectPage project={project} allProjects={projects} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
