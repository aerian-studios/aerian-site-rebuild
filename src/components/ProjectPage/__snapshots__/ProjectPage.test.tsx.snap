// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectPage renders correctly 1`] = `
Array [
  <header
    className="block block--full layout-grid component"
    id="page-header"
    style={Object {}}
  >
    <figure
      className="undefined undefined"
    >
      <div
        className=""
      >
        <img
          src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/280/bbc-perfect-diet-for-you-hero-imagejpg.jpg"
        />
      </div>
    </figure>
    <div>
      <h1>
        What sort of 'eater' are you?
        <br />
        I mean... really?!
      </h1>
    </div>
  </header>,
  <nav
    className=""
  >
    <div
      className=""
    >
      <a
        className=""
        href="/#caseStudy"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Introduction
      </a>
      <a
        className=""
        href="/#gallery"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Gallery
      </a>
      <a
        className=""
        href="/#challenge"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        The Challenge
      </a>
      <a
        className=""
        href="/#solution"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        The Solution
      </a>
      <a
        className=""
        href="/#results"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        The Result
      </a>
    </div>
  </nav>,
  <div
    className=" "
  >
    <div>
      Client logo
    </div>
    <div
      className=" "
    >
      <h3>
        Case study
      </h3>
      <h2>
        Two interactive 'foodie' quizzes for BBC iWonder
      </h2>
      <div>
        <p>
          With the launch of a new BBC1 series, we created two fun interactive quizzes that shine a light on all our weird and awkward eating habits.
        </p>
      </div>
      <a
        href="http://www.bbc.co.uk/guides/z2csfg8"
      >
        Launch project
      </a>
    </div>
  </div>,
  <div
    className=" "
  >
    <div
      className=" "
    >
      <div
        className=""
      >
        <img
          alt="BBC perfect diet for you grid image 1.jpg"
          src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/383/bbc-perfect-diet-for-you-grid-image-1.jpg.jpg"
        />
      </div>
      <div
        className=""
      >
        <img
          alt="BBC perfect diet for you grid image 2.jpg"
          src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/384/bbc-perfect-diet-for-you-grid-image-2.jpg.jpg"
        />
      </div>
      <div
        className=""
      >
        <img
          alt="BBC perfect diet for you grid image 3.jpg"
          src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/385/bbc-perfect-diet-for-you-grid-image-3.jpg.jpg"
        />
      </div>
      <div
        className=""
      >
        <img
          alt="bbc-perfect-diet-for-you-grid-image-4"
          src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/467/bbc-perfect-diet-for-you-grid-image-4.jpg"
        />
      </div>
    </div>
  </div>,
  <div
    className=" "
  >
    <div
      className=" "
    >
      <div
        className=" "
      >
        <h2>
          The Challenge
        </h2>
        <p>
          Whatever we built needed to be awesome, no matter how non-awesome someone's device might be
        </p>
      </div>
      <div
        className=" "
      >
        <div
          className=" "
        >
          <div
            className=""
          >
            <img
              alt="Responsive guidelines "
              src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/622/bbc-iwonder-01.png"
            />
          </div>
          <h3>
            Responsive guidelines 
          </h3>
          <div>
            <ul>
              <li>
                Extensive UX and design reviews would require special attention to detail so that the final product would meet BBC needs.
              </li>
              <li>
                Work with iWonder responsive guidelines.
              </li>
            </ul>
          </div>
        </div>
        <div
          className=" "
        >
          <div
            className=""
          >
            <img
              alt="Audience sharing "
              src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/623/bbc-iwonder-02.png"
            />
          </div>
          <h3>
            Audience sharing 
          </h3>
          <div>
            <ul>
              <li>
                Work with iWonder games guidelines.
              </li>
              <li>
                Creating an app framework that will work with a variety of question types.
              </li>
              <li>
                High audience and sharing figures were a requirement.
              </li>
            </ul>
          </div>
        </div>
        <div
          className=" "
        >
          <div
            className=""
          >
            <img
              alt="Tight turn around"
              src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/624/bbc-iwonder-03.png"
            />
          </div>
          <h3>
            Tight turn around
          </h3>
          <div>
            <ul>
              <li>
                As always, a tight turn around time.
              </li>
              <li>
                The total download size of the initial activity load MUST be less than 250kb.
              </li>
              <li>
                Any individual asset within an activity SHOULD NOT exceed 100KB.
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className=" "
  >
    <div
      className=" "
    >
      <div
        className=" "
      >
        <h2>
          The Solution
        </h2>
        <p>
          We used highly flexible design solutions that utilised device interactions such as advanced swipe motions and parallax via CCS3, JavaScript and HTML5, thus reaching the largest device demographic possible. 
        </p>
      </div>
      <div
        className=" "
      >
        <div
          className=" "
        >
          <div
            className=""
          >
            <img
              alt="Maximising sharing "
              src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/625/bbc-iwonder-04.png"
            />
          </div>
          <h3>
            Maximising sharing 
          </h3>
          <div>
            <p>
              To increase the appeal of the app, animated features and interactions were added. Share buttons situated at key stages of the journey maximised exposure of sharing.
            </p>
          </div>
        </div>
        <div
          className=" "
        >
          <div
            className=""
          >
            <img
              alt="Agile approach"
              src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/626/bbc-iwonder-05.png"
            />
          </div>
          <h3>
            Agile approach
          </h3>
          <div>
            <ul>
              <li>
                Agile project approach to ensure all progress and blockers are expedited.
              </li>
              <li>
                Close working relationship with BBC to ensure guidelines were adhered.
              </li>
            </ul>
          </div>
        </div>
        <div
          className=" "
        >
          <div
            className=""
          >
            <img
              alt="CMS implementation"
              src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/627/bbc-iwonder-06.png"
            />
          </div>
          <h3>
            CMS implementation
          </h3>
          <div>
            <p>
              Content management was implemented so that content and javascript elements were editable.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className=" "
  >
    <div
      className=" "
    >
      <div
        className=" "
      >
        <h2>
          The Result
        </h2>
        <p>
          With over 1.5 million unique users in 48 hours, it quickly became the number one iWonder guide of all time, beating even the 2014 World Cup! 
        </p>
      </div>
      <div
        className=" "
      >
        <div
          className=" "
        >
          <div
            className=""
          >
            <img
              alt="Furthest reaching guide"
              src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/628/bbc-iwonder-07.png"
            />
          </div>
          <h3>
            Furthest reaching guide
          </h3>
          <div>
            <p>
              "The team at Aerian were a pleasure to work with from start to finish, and helped BBC iWonder deliver its furthest-reaching iWonder guide and interactive build to date, the 'What's the right diet for you?' quiz."
            </p>
          </div>
        </div>
        <div
          className=" "
        >
          <div
            className=""
          >
            <img
              alt="Seen by 1.5m"
              src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/629/bbc-iwonder-08.png"
            />
          </div>
          <h3>
            Seen by 1.5m
          </h3>
          <div>
            <p>
              It was seen by 1.5m unique browsers and was really well received by audience members.
            </p>
          </div>
        </div>
        <div
          className=" "
        >
          <div
            className=""
          >
            <img
              alt="Successful weightloss "
              src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/630/bbc-iwonder-09.png"
            />
          </div>
          <h3>
            Successful weightloss 
          </h3>
          <div>
            <p>
              "Users wrote in to notify us of their success losing weight having taken the test, and then the onward journey to BBC Food recipes. We couldn't have done it without Aerian." Anna-Louise Taylor, BBC Senior Content Producer
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className=" "
  >
    <div
      className=" "
      itemProp="review"
      itemScope={true}
      itemType="http://schema.org/Review"
    >
      <div
        className=""
      >
        <img
          src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/281/bbc-perfect-diet-for-you-qoute-image.jpg.jpg"
        />
      </div>
      <blockquote>
        <i
          className="fa"
        />
        <div
          itemProp="reviewBody"
        >
          It was seen by 1.5m unique browsers and was really well received by audience members. Users wrote in to notify us of their success losing weight having taken the test, and then the onward journey to BBC Food recipes. We couldn’t have done it without Aerian.
        </div>
        <footer>
          <figure>
            <div
              className=""
            >
              <img
                src="https://raw.githubusercontent.com/aerian-studios/aerian-site-rebuild/master/static/assets/imported/282/anna.png"
              />
            </div>
          </figure>
          <div>
            <span
              itemProp="author"
            >
               Anna-Louise Taylor
            </span>
            <span
              itemProp="name"
            >
              Senior Content Producer
            </span>
          </div>
        </footer>
      </blockquote>
    </div>
  </div>,
  <div
    className=" "
  >
    <div
      className=" "
    >
      <h2>
        Performance & Statistics
      </h2>
      <div
        className=" "
      >
        <div>
          <h4>
            558000
          </h4>
          <p>
            558,000 additional visits
          </p>
        </div>
        <div>
          <h4>
            240000
          </h4>
          <p>
            240,000 visits
          </p>
        </div>
        <div>
          <h4>
            1500000
          </h4>
          <p>
            1,500,000+ visits
          </p>
        </div>
      </div>
    </div>
  </div>,
  <div
    className=" "
  >
    <div
      className=" "
    >
      <div>
        <h2>
          Like what you see?
        </h2>
        <a
          href="http://www.bbc.co.uk/guides/z2csfg8"
        >
          Launch project
        </a>
      </div>
      <div>
        <h2>
          Got a project in mind?
        </h2>
        <a
          href="/contact-us"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Contact us
        </a>
      </div>
    </div>
  </div>,
]
`;
